### ISSUE TITLE
Add support for DCLS and BDEF object types

### ISSUE THREAD

**Issue by Fxlipe115** [abapinho/abapTimeMachine#119]
It would be very useful to have access to abapblame's functionality also on CDSs and on ABAP RAP's behavior definitions.

**Comment by nununo** [comment]
Good idea. I'll look into it as soon as I have the time (I've been very busy lately). Thanks!
